lnrpc/rpc.proto

```diff
@@ -1428,6 +1428,14 @@ message Channel {
     cooperatively closing with the delivery_address field set.
     */
     string close_address = 25;
+
+    /**
+    This uint32 indicates if this channel is to be considered 'frozen'. A
+    frozen channel doest not allow a cooperative channel close by the
+    initiator. The thaw_height is the height that this restriction stops
+    applying to the channel.
```

Comment:
We should either mention it's optional or explicitly state that leaving it at value 0 will have no effect.